meta-facebook/yv4-sd/src/platform/plat_mctp.c

```diff
@@ -0,0 +1,326 @@
+/*
```

Comment:
The mctp code seems to be generic and better to keep it in common layer, so other platforms can also use it.

Reply:
It is a lot more complicate to put the code in common layer, since every platform is using the same common code. We need to make sure that ongoing projects won't be affected. Therefore, we plan to put this on platform like other project in this stage.

Reply:
@PeterHo-wiwynn can you move it to common and change meta-facebook/gt-cc/src/platform/plat_mctp.c as well. I can help with testing GTT/GTI which are the only other platforms which use this.

Preferably I would like to see plat_*.c contain just configuration data structures not functions which do actions.

Reply:
Sure, I will move it to common. However, there are still some functions which implemented differently between platforms. I will try to move the functions doing same behavior, and list the others for disscusion.